packages/jest-haste-map/src/index.ts

```diff
@@ -996,6 +1002,7 @@ class HasteMap extends EventEmitter {
   }
 
   end(): Promise<void> {
+    // @ts-ignore: TODO TS cannot decide if `setInterval` and `clearInterval` comes from NodeJS or the DOM
```

Comment:
this is super weird - `setInterval` returns `NodeJS.Timeout`, but `clearInterval` expects `number`. Not sure what's up?